src/mapping/RadialBasisFctMapping.hpp

```diff
@@ -212,7 +219,7 @@ void RadialBasisFctMapping<RADIAL_BASIS_FUNCTION_T>::mapConservative(DataID inpu
 
     // Construct Eigen vectors
     Eigen::Map<Eigen::VectorXd> inputValues(globalInValues.data(), globalInValues.size());
-    Eigen::VectorXd             outputValues((_rbfSolver.getEvaluationMatrix().cols() - this->getPolynomialParameters()) * valueDim);
+    Eigen::VectorXd             outputValues((this->output()->getGlobalNumberOfVertices()) * valueDim);
```

Comment:
I was not sure whether it is a good idea to rely on `getGlobalNumberOfVertices`. I needed to add the information in all places in the RBF unit tests. Motivation was here to get rid of the dependency of the polynomial.

Reply:
It should give the correct value AFAIK. Alternative could be to store this in `_rbfSolver`.